lib/ocpp/v201/charge_point.cpp

```diff
@@ -792,7 +792,7 @@ std::optional<std::string> ChargePoint::get_evse_transaction_id(int32_t evse_id)
     return evse.get_transaction()->transactionId.get();
 }
 
-AuthorizeResponse ChargePoint::validate_token(const IdToken id_token, const std::optional<CiString<5500>>& certificate,
+AuthorizeResponse ChargePoint::validate_token(const IdToken id_token, const std::optional<CiString<10000>>& certificate,
```

Comment:
TODO: find out implications for backwards compatibility with 2.0.1

Reply:
accept